src/client/pages/ToeristischeVerhuurDetail/ToeristischeVerhuurDetail.tsx

```diff
@@ -50,11 +51,9 @@ export default function ToeristischVerhuurDetail() {
         }}
         isLoading={isLoading(TOERISTISCHE_VERHUUR)}
       >
-        {getHeaderTitle(
-          Vergunning?.caseType ?? '',
-          Vergunning?.dateStart ?? '',
-          Vergunning?.isPast
-        )}
+        {Vergunning
```

Comment:
Deze ternary is niet nodig, de `getHeaderTitle` functie heeft een default 'Onbekende Toeristische verhuur' wanneer het caseType niet bepaald kan worden. Zodoende staat er altijd een titel.